### PR TITLE
[CAZB-2699] Change "Back" button to use web-browser native feature

### PR DESCRIPTION
**Pull request by jenniferbrown94**
https://eaflood.atlassian.net/browse/CAZB-2699

Updating all back buttons to use JS - this is to be merged in so that QA team can test the impact of this, further work will be done to ensure that data that is currently stored in the session can be accessed correctly on each page.

## Checklist:

- [ ] It contains only changes required by issue (does not contain other PR)
- [ ] Includes the link to an issue (if apply)
- [ ] I have added tests to cover my changes.

### CODE REVIEW

File: app/javascript/src/backButton.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import "jquery"
+
+window.onload = function () {
+    $("#back").removeClass('visually-hidden')
+    $("#back").removeClass('aria-hidden')
+
+    $("#back").click(function(event) {
```
Comment (dratka) [inline app/javascript/src/backButton.js:7]: for the consistency we could use the same script as we have in other services:
https://github.com/InformedSolutions/JAQU-CAZ-Customer-Service-UI/pull/250

File: app/views/charges/local_authority.html.haml [hunk 2]
```diff
@@ -1,5 +1,5 @@
 - page_title('Which Clean Air Zone do you need to pay for?')
-= link_to 'Back', @return_path, class: 'govuk-back-link', id: 'back-link'
+= link_to 'Back', "", class: 'govuk-back-link visually-hidden aria-hidden', id: 'back'
```
Comment (dratka) [inline app/views/charges/local_authority.html.haml:2]: Can we move the back links to partial? 
Example here: https://github.com/InformedSolutions/JAQU-CAZ-Customer-Service-UI/pull/250/files

File: app/javascript/src/backButton.js (hunk 1 above)
Comment (jenniferbrown94) [reply app/javascript/src/backButton.js]: thanks for the feedback - this should now be updated.

File: app/javascript/src/backLink/init.js [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+    if (backLink) {
+      backLink.style.display = 'inline-block';
+      backLink.addEventListener('click', (event) => {
+        event.preventDefault();
+        history.back();
+      }, false);
+    }
+  }
+  
+  export default init;
```
Comment (pkalamarski) [inline app/javascript/src/backLink/init.js:12]: Inconsistent indentation, lines 2-12 should be moved 2 spaces back

Comment (pkalamarski) [inline app/javascript/src/backLink/init.js:12]: Missing newline at end of file

File: app/views/common/_js_back_link.html.haml [hunk 4]
```diff
@@ -0,0 +1,2 @@
+%a.govuk-back-link#js-back-link{href: '#', style: 'display: none;'}
+  Back
```
Comment (pkalamarski) [inline app/views/common/_js_back_link.html.haml:2]: Missing newline at end of file

**Pull request merged**